UnitsNet.Tests/CustomCode/DensityTests.cs

```diff
@@ -23,10 +23,14 @@ public class DensityTests : DensityTestsBase
 
         protected override double KilogramsPerCubicMillimeterInOneKilogramPerCubicMeter => 1e-9;
 
+        protected override double KilopoundsPerCubicYardInOneKilogramPerCubicMeter => 1.6855549356e-3;
+
         protected override double KilopoundsPerCubicFootInOneKilogramPerCubicMeter => 6.242796e-5;
 
         protected override double KilopoundsPerCubicInchInOneKilogramPerCubicMeter => 3.6127292e-8;
 
+        protected override double PoundsPerCubicYardInOneKilogramPerCubicMeter => 1.6855549356;
```

Comment:
Verified 

`0.45359237 kg ÷ ( 0.9144 yd ^ 3 ) = 1.685554935555904522820787307335` 


1.6855549: https://www.convert-me.com/en/convert/density/lbpyd/lbpyd-to-kgpm.html?u=kgpm&v=1